src/applications/rx/containers/RxRefillsApp.jsx

```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import backendServices from '../../../platform/user/profile/constants/backendServices';
 import DowntimeNotification, { externalServices } from '../../../platform/monitoring/DowntimeNotification';
 import MHVApp from '../../../platform/user/authorization/containers/MHVApp';
-import Breadcrumbs from '../components/Breadcrumbs';
+import RxBreadcrumbs from '../components/Breadcrumbs';
```

Comment:
If we are planning to name the breadcrumb components in each app, should we consider using the same name as the 'App' component, such as RxRefillsApp (so RxRefillsBreadcrumbs)?